ops/model.py

```diff
@@ -817,6 +825,9 @@ def action_log(self, message):
     def action_fail(self, message=''):
         self._run('action-fail', message)
 
+    def application_version_set(self, version):
```

Comment:
I chose to call the backend the same as the actual command we're calling. I changed the name for Unit.set_workload_version because that is what we want people to interact with. Another option is that ModelBackend would do the abstraction, but for now it seems better to have it be the obvious matching.